src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Loader } from '../Loader/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const { slug = '' } = useParams();
```

Comment:
It's better to use a more specific type for the useParams hook. You can define an interface for the expected parameters and pass it to the hook like this: `useParams<YourInterface>();`